src/coffee/connect/rest.coffee

```diff
@@ -193,7 +193,7 @@ class Rest
     throw new Error 'Query limit doesn\'t seem to be 0. This function queries all results, are you sure you want to use this?' if query.limit and query.limit isnt '0'
 
     params = _.extend {}, query,
-      limit: 50 # limit used for batches
+      limit: 500 # limit used for batches
```

Comment:
Actually this method is used only for `all().fetch()`. There's also a comment at the beginning of this method:

`Public: Fetch all results of a Sphere resource query endpoint in batches of pages using a recursive function`